pennylane/interfaces/batch/autograd.py

```diff
@@ -42,6 +42,10 @@ def execute(tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1):
         gradient_fn (callable): the gradient function to use to compute quantum gradients
         _n (int): a positive integer used to track nesting of derivatives, for example
             if the nth-order derivative is requested.
+        max_diff (int): If ``gradient_fn`` is a gradient transform, this option specifies
+            the maximum number of derivatives to support. Increasing this value allows

```

Comment:
Is this maximum _number_ or maximum _order_ of derivatives?

Reply:
```suggestion
            the maximum order of derivatives to support. Increasing this value allows
```